4.analyze_data/notebooks/linear_model/0.fit_linear_model_strat.ipynb

```diff
@@ -19,7 +19,7 @@
     "from sklearn.linear_model import LinearRegression\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        lm = LinearRegression(fit_intercept=True)
</pre><p>Based on the source code, the <code>fit_intercept=True</code> by default:</p><p><a href="https://github.com/scikit-learn/scikit-learn/blob/093e0cf14aff026cca6097e8c42f83b735d26358/sklearn/linear_model/_base.py#L640C10-L640C10" target="_blank">https://github.com/scikit-learn/scikit-learn/blob/093e0cf14aff026cca6097e8c42f83b735d26358/sklearn/linear_model/_base.py#L640C10-L640C10</a>  </p><p> </p><p>I would recommend removing it unless there's a need for explicitly stating it.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/CFReT_data/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Great catch! I will remove it since there isn't a specific reason to state it.